index.html

```diff
@@ -1391,6 +1391,9 @@
         </dt>
         <dd>
           A <a>valid decimal monetary value</a> containing a monetary amount.
+          User agents should not alter values provided as input; for example,
```

Comment:
As I mentioned in the bug, it's a "statement of fact" that UAs can present the value differently - but this doesn't belong here. It belongs in the "currency member" description. 

I suggest we add the following, underneath  "A user agent SHOULD use U+00A4 (¤) for formatting.": 

```
Users agents MAY format the display of the <a>currency</a> member 
to adhere to OS conventions (e.g., for localization purposes).
```

Reply:
Ok, but this should be removed now :) We definitely don't want this SHOULD here.

Reply:
Hi @marcoscaceres,

Do you mean the same sentence belongs somewhere else? That would be fine. But I hope we get to keep it or something close.

Ian

Reply:
No, I mean it should be removed entirely. The algorithms already handle all this stuff... saying "...SHOULD NOT..." here would lead to confusion.

Reply:
To be clear, if we were to leave the "SHOULD NOT", it opens the door to the possibility that an implementer could change the value... that would be very very bad and would lead to all sorts of interop chaos. 

Instead, implementers should follow what the algorithms says - which make it clear, if you follow them, that the value never gets changed.

Reply:
I think it would be helpful to include an explicit note to that effect. I will propose something in an updated PR.